biocrnpyler/construct_explorer.py

```diff
@@ -98,7 +97,7 @@ def terminate_loop(self):
         pass
         #MUST SUBCLASS
 
-    def return_components(self,component):
+    def return_components(self,component, previously_enumerated = None):
```

Comment:
because the construct explorer doesn't normally know what construct it is going to be exploring. It's sort of like how mechanisms take a component as input even though they belong to a component